packages/clay-form/src/MultiSelect.tsx

```diff
@@ -0,0 +1,319 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import React, {useLayoutEffect, useRef, useState} from 'react';
+
+const COMMA = 188;
+
+const KEYS = [COMMA];
+
+const BACKSPACE_KEY = 8;
+
+export interface IProps extends React.HTMLAttributes<HTMLInputElement> {
+	/**
+	 * List of key codes that will trigger onItemsChange(inputValue).
+	 */
+	deliminatorKeys?: number[];
```

Comment:
I don't think "deliminator" is a word; would "delimiterKeys" or "separatorKeys" be better?

Reply:
"Delimiter" is what I was going for lol